java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -467,6 +473,8 @@ private boolean handleMethodInvocation(Instruction instruction) {
             }));
         return true;
       }
+    } else {
+      // TODO remove "thisSV" from stack before trying to apply any yield, as it should not match with arguments
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)